### ISSUE TITLE
code:snake

### ISSUE THREAD

**Issue by egonaut** [XRadius/http-game-apex#2]
First off, great work here. Just wanted to report that Sense caused code:snake on a fresh account (and fresh hardware/a fresh network, actually) so something is clearly getting picked up by EAC. Typically code:snake is what appears before a temp ban but it could also be permanent. 

Instead of displaying the players by directly modifying the memory, would it be possible to get the information related to the browser via http-driver (ie. first person view that shows location of enemies as you look around, similar to how Radar works but just 3D)

I believe directly modifying the memory via Sense is what is causing code:snake, and in my case, was after two games.

**Comment by egonaut** [comment]
Also, this occurred while playing on Arch, not a deb based distro. Configuration was followed correctly and adjusted for Arch as opposed to Ubuntu.

**Comment by XRadius** [comment]
Yes, this is possible. But there is pretty big downside to that. Plotting something on top of the game screen requires an overlay, and in Linux, the GUI subsystem is not user-isolated. That means that it's quite trivial to detect the overlay. The one remaining solution is to use a multi-machine setup and a game capture card, but that introduces latency. Neither option is very appealing. In contrast, since Epic doesn't seem to collaborate with EA for game-specific checks, the modifications Sense applies are *relatively safe*. Right now, I feel that this is the best approach. That said, I have a branch with a 3D-version of the radar I'm not actively working on. If you're interested, I'll push the branch so you can continue the work on it.
